evap/staff/tests/test_views.py

```diff
@@ -327,7 +402,7 @@ def test_invalid_upload_operation(self):
         form = page.forms["user-import-form"]
         page = form.submit(name="operation", value="test")
 
-        self.assertContains(page, 'Please select an Excel file')
+        self.assertContains(page, 'This field is required.')
```

Comment:
I did not check myself, but would it be possible to check `form.errors`, so that we also check that the message is also on the correct field?
(Also applies to later occurrences)